CITATION.cff

```diff
@@ -0,0 +1,69 @@
+cff-version: 1.2.0
+message: "If you use MFEM, please cite it as follows."
+authors:
+- family-names: "MFEM Team"
+title: "MFEM: Modular Finite Element Methods [Software]"
+doi: 10.11578/dc.20171025.1248
+url: "https://mfem.org"
+preferred-citation:
+  type: article
+  authors:
+  - family-names: "Anderson"
+    given-names: "Robert"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Andrej"
+    given-names: "Julian"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Barker"
+    given-names: "Andrew"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Bramwell"
+    given-names: "Jamie"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Camier"
+    given-names: "Jean-Sylvain"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Cerveny"
+    given-names: "Jakub"
+    orcid: "https://orcid.org/0000-0003-4231-2531"
+  - family-names: "Dobrev"
+    given-names: "Veselin"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Dudouit"
+    given-names: "Yohann"
+    orcid: "https://orcid.org/0000-0001-5831-561X"
+  - family-names: "Fisher"
+    given-names: "Aaron"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Kolev"
+    given-names: "Tzanio"
+    orcid: "https://orcid.org/0000-0002-2810-3090"
+  - family-names: "Pazner"
+    given-names: "Will"
+    orcid: "https://orcid.org/0000-0003-4885-2934"
+  - family-names: "Stowell"
+    given-names: "Mark"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Tomov"
+    given-names: "Vladimir"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Akkerman"
+    given-names: "Ido"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Dahm"
+    given-names: "Johann"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Medina"
+    given-names: "David"
+    orcid: "https://orcid.org/0000-0000-0000-0000"
+  - family-names: "Zampini"
+    given-names: "Stefano"
+    orcid: "https://orcid.org/0000-0002-0435-0433"
+  doi: "10.1016/j.camwa.2020.06.009"
+  journal: "Computers and Mathematics with Applications"
```

Comment:
```suggestion
  journal: "Computers & Mathematics with Applications"
```
We should check that this doesn't mess up the BibTeX rendering.

Reply:
Try it here: https://github.com/tzanio/mfem

Reply:
With `Computers & Mathematics with Applications` the BibTeX is:

```bib
@article{Anderson_MFEM_A_Modular_2021,
author = {Anderson, Robert and Andrej, Julian and Barker, Andrew and Bramwell, Jamie and Camier, Jean-Sylvain and Cerveny, Jakub and Dobrev, Veselin and Dudouit, Yohann and Fisher, Aaron and Kolev, Tzanio and Pazner, Will and Stowell, Mark and Tomov, Vladimir and Akkerman, Ido and Dahm, Johann and Medina, David and Zampini, Stefano},
doi = {10.1016/j.camwa.2020.06.009},
journal = {Computers & Mathematics with Applications},
month = {1},
pages = {42--74},
title = {{MFEM: A Modular Finite Element Library}},
volume = {81},
year = {2021}
}
```

With `Computers \& Mathematics with Applications` I get an error.

Maybe there is another way of doing it?

Reply:
Yeah, it doesn't escape the `&` unfortunately ☹️ 

`Computers \& Mathematics with Applications` should work fine (it does for me)

Reply:
Like this: https://github.com/tzanio/mfem/blob/master/CITATION.cff ?

Reply:
I got it... double escape

Reply:
Oh, sorry, I misunderstood. `\&` works for the BibTeX, but I couldn't get it to work with the `cff` file.

Lets just leave it as it is then 🙂

Reply:
it is fixed :)

Reply:
Smart!!